gf-ui-lib/src/components/Map/MapContext.ts

```diff
@@ -1,10 +1,22 @@
 import {createContext} from "react";
-import {IMarker} from "./mapContent/marker/IMarker";
+import {Marker} from "./mapLeaflet/mapMarkers/mapMarker/Marker";
 
-export interface IMapContext {
-  markers: IMarker[];
-  menuItem: IMarker | null;
-  setMenuItem: React.Dispatch<React.SetStateAction<IMarker | null>>;
+/**
+ * Map context
+ */
+export interface MapContext {
+  /**
+  * Markers list
+  */
+  markers: Marker[];
+  /**
+  * Selected menu item
+  */
+  menuItem: Marker | null;
+  /**
+  * Sets the selected menu item
+  */
+  setMenuItem: React.Dispatch<React.SetStateAction<Marker | null>>;
 }
 
-export const MapContext = createContext<IMapContext>({} as IMapContext);
\ No newline at end of file
+export const MapContext = createContext<MapContext>({} as MapContext);
```

Comment:
Need to see other Contexts and avoid construction as ...

Reply:
Done